app/src/main/java/org/oppia/android/app/resumelesson/ResumeLessonActivity.kt

```diff
@@ -87,6 +90,7 @@ class ResumeLessonActivity :
     parentScreen: ExplorationActivityParams.ParentScreen,
     isCheckpointingEnabled: Boolean
   ) {
+    resumeLessonActivityPresenter.setReadingTextSizeNormal()
```

Comment:
I believe there is no need to reset the size here.

Reply:
If we don't set text size medium it will affect the exploration tool bar title text size.

[Resume.webm](https://github.com/oppia/oppia-android/assets/76042077/1615b7e0-fdfb-4235-8f43-46c91c22fb11)